neofetch

```diff
@@ -11435,6 +11438,33 @@ ${c3}#######${c2}#${c1}#####${c2}#${c3}#######
 ${c3}  #####${c2}#######${c3}#####
 EOF
                 ;;
+
+"VzLinux"*)
+set_colors 1 7 3
+read -rd '' ascii_data <<'EOF'                                            
```

Comment:
This shouldn't be added here, it should be added after VNux. This part of the script is for kernel name detection when distro name detection fails.